README.md

```diff
@@ -138,6 +138,14 @@
 
 KNOWN ISSUE'S: The current OAUTH plugin is not up to date (2.11.3) test or compile the latest version from the website: https://github.com/davido/gerrit-oauth-provider
 
+## Using gitiles instead of gitweb
+docker run \
+    --name gerrit \
+    -p 8080:8080 \
+    -p 29418:29418 \
+    -e GITWEB_TYPE=gitiles \
+    -d openfrontier/gerrit
+
```

Comment:
These lines seems not rendering correctly by github's markdown. You might need to indent the `docker run` line.